apidoc/Titanium/Android/NotificationManager/NotificationManager.yml

```diff
@@ -1,87 +1,19 @@
 ---
 name: Titanium.Android.NotificationManager
-summary: Module for managing status bar notifications.
+summary: Module for managing notifications.
 description: |
-    This module provides methods for sending and canceling status bar notifications,
+    This module provides methods for sending and canceling notifications,
     which are represented by the <Titanium.Android.Notification> object.
 
-    The `NotificationManager` module is a thin wrapper on the Android `NotificationManager`
-    class. The constants in this module are derived from the Android `Notification` class.
+    Further Reading:
 
-    See also:
-
-    *   [Android Notifications in the Titanium Mobile Guides](http://docs.appcelerator.com/titanium/2.0/#!/guide/Android_Notifications)
-
-    *   [Status Bar Notifications in the Android Developer Guide](http://developer.android.com/guide/topics/ui/notifiers/notifications.html)
-
-    *   [NotificationManager in the Android API Reference](http://developer.android.com/reference/android/app/NotificationManager.html) 
-
-    *   [Notification in the Android API Reference](http://developer.android.com/reference/android/app/Notification.html) 
+      * [Android Notifications guide](http://docs.appcelerator.com/titanium/latest/#!/guide/Android_Notifications)
+      * [Android Developer: Notifications](http://developer.android.com/guide/topics/ui/notifiers/notifications.html)
 
 extends: Titanium.Module
 platforms: [android]
 since: "1.5"
 
-examples: 
```

Comment:
Still there.  Just moved it to the Ti.Android.Notification doc where I thought it would be more useful.